FlashFeetAPI/src/main/java/org/example/resources/JobRoleController.java

```diff
@@ -0,0 +1,61 @@
+package org.example.resources;
+
+import io.swagger.annotations.Api;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.StreamingOutput;
+
+import org.example.api.JobRoleService;
+import org.example.client.FailedToDeleteJobRoleException;
+import org.example.client.FailedToGetJobsException;
+import org.example.client.JobRoleDoesNotExistException;
+
+import org.example.db.DatabaseConnector;
+
+import java.sql.SQLException;
+
+
+@Api("FlashFeet Kainos Job Data API")
+@Path("/api")
+public class JobRoleController {
+    private JobRoleService jobRoleService = new JobRoleService();
+
+    DatabaseConnector databaseConnector = new DatabaseConnector();
+
+    @GET
+    @Path("/job-role-list")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getAllJobRoles(){
+        System.out.println("Test");
+        try {
+            return Response.ok(jobRoleService.getAllJobRoles()).build();
+        } catch (FailedToGetJobsException e){
```

Comment:
I feel it would be best to combine the SQLException here and return 500 error? An even better way would be to handle your SQLException closer to the DAO level. Possibly in the service?